samples/Http2SampleApp/scripts/run-docker.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
```

Comment:
Nit: DIR isn't used.

Reply:
True, I'll remove it from this. Just copy 🍝